py3status/modules/yubikey.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    format: Display format for the module.
+        (default '[YubiKey[\?if=is_gpg ][\?if=is_u2f ]]')
+    socket_path: A path to the yubikey-touch-detector socket file.
+        (default '$XDG_RUNTIME_DIR/yubikey-touch-detector.socket')
+
+Control placeholders:
+    {is_gpg} a boolean, True if YubiKey is waiting for a touch due to a gpg operation.
+    {is_u2f} a boolean, True if YubiKey is waiting for a touch due to a pam-u2f request.
+
+Requires:
+    github.com/maximbaz/yubikey-touch-detector: tool to detect when YubiKey is waiting for a touch
```

Comment:
(Nit) Three lines here are over column 80. I don't have 4K monitors.  ;__;

Reply:
the line length is enforced by Travis, it was long but I compressed it enough so that Travis is green and happy now.

you should by a 4K screen, and a yubikey :stuck_out_tongue_winking_eye: